### PR TITLE
Update web page

### PR DESCRIPTION
**Pull request by daniel-otero**

### CODE REVIEW

File: src/css/custom.css [hunk 1]
```diff
@@ -22,28 +22,35 @@
...
+  --ifm-color-primary-light: #c0abff;
+  --ifm-color-primary-lighter: #cebeff;
+  --ifm-color-primary-lightest: #faf8ff;
+  --ifm-background-color: #1b1725 !important; /* Otherwise they are overwritten by html[data-theme='dark'] */
+  --ifm-background-surface-color: #32324a !important; /* Otherwise they are overwritten by html[data-theme='dark'] */
   --docusaurus-highlighted-code-line-bg: rgba(0, 0, 0, 0.3);
 }
 
+footer {
+  --ifm-footer-background-color: #32324a !important;
```
Comment (alvrogd) [inline src/css/custom.css:51]: I understand this `!important` is needed for the same reason as those above.

**Review by alvrogd — APPROVED** [review]
Looking good! Could you include a brief description for the PR as well? Nothing super detailed, it's enough to mention that these are some adjustments after the previous PR that added the web.

File: src/css/custom.css (hunk 1 above)
Comment (daniel-otero) [reply src/css/custom.css]: The important wasn't really needed, as I can choose a more specific class to override the value. Not sure which approach is best, TBH.

Thanks for pointing it out, anyway.

Comment (alvrogd) [reply src/css/custom.css]: I believe choosing a more specific selector is considered to be the best-practice. But either approach is fine with me.

File: docusaurus.config.ts [hunk 2]
```diff
@@ -83,23 +83,37 @@ const config: Config = {
...
           title: 'Community',
           items: [
+            {
+              label: 'LinkedIn',
+              href: 'https://www.linkedin.com/company/codee-com/',
+            },
             {
               label: 'Twitter',
-              href: 'https://twitter.com/codee_com',
+              href: 'https://x.com/codee_com/',
```
Comment (ruifm) [inline docusaurus.config.ts:94]: label should be `X`

**Review by ruifm — APPROVED** [review]

**Comment by daniel-otero** [comment]
The branch was merged, but seems like I forgot to update the PR and wasn't synced correctly.

**Pull request closed without merging**